pkg/io/storage/storage_test.go

```diff
@@ -0,0 +1,57 @@
+package storage
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestFileVault_SaveData(t *testing.T) {
+
+	fileVault, err := NewFileVault("file")
+	defer func() {
```

Comment:
Too early to close `fileVault` since error could be returned and `fileVault == nil` which results in a nil dereference.